### PR TITLE
Bug 1866693: docs: steps to enable RHCOS extensions during cluster install

### PR DESCRIPTION
**Pull request by sinnykumari**

### CODE REVIEW

**Comment by runcom** [comment]
cc @yuqi-zhang @ericavonb @cgwalters

**Comment by runcom** [comment]
lgtm overall also! thanks for this Sinny, I'll leave to the rest of the team to fully approve

File: docs/user/customization.md [hunk 1]
```diff
@@ -511,6 +517,30 @@ Example for switching to RT kernel on worker nodes during initial cluster instal
...
+  name: worker-extensions
+spec:
+  config:
+    ignition:
+      version: 3.1.0
+  extensions:
+    - usbguard
+  ```
+
+**Note:** `kernel-devel` extension should be installed only when you are using compatible traditional kernel (kernleType: default).
```
Comment (yuqi-zhang) [inline docs/user/customization.md:542]: minor typo: kernleType -> kernelType

Comment (sinnykumari) [reply docs/user/customization.md]: fixed

**Comment by sinnykumari** [comment]
can someone please approve the PR, thanks!

**Pull request merged**